stubs/ExifRead/exifread/utils.pyi

```diff
@@ -0,0 +1,15 @@
+from _typeshed import Incomplete
+from fractions import Fraction
+
+def ord_(dta): ...
+def make_string(seq: bytes | list[Incomplete]) -> str: ...
+def make_string_uc(seq) -> str: ...
+def get_gps_coords(tags: dict[Incomplete, Incomplete]) -> tuple[Incomplete, Incomplete]: ...
```

Comment:
```suggestion
def get_gps_coords(tags: Mapping[Incomplete, Incomplete]) -> tuple[Incomplete, Incomplete]: ...
```

Reply:
I'm preparing a followup PR now and will include this there. Thanks!